code/game/objects/items/candle.dm

```diff
@@ -74,11 +74,14 @@
 	return 0
 
 /obj/item/candle/Crossed(var/obj/Proj)
-	if(isbeam(Proj))
-		//Can also be an obj/effect, but they both have the damage var
+	if(istype(Proj,/obj/item/projectile/beam))
```

Comment:
Why did you change this helper ?

Reply:
Beams no longer have the damage var, gotta give projectiles a get_damage proc instead. I think this bit of code was risky anyway to assume that both types would always share that same var.